services/core-web/src/components/mine/Tailings/MineTailingsTable.tsx

```diff
@@ -148,37 +206,31 @@ const MineTailingsTable: FC<RouteComponentProps & MineTailingsTableProps> = (pro
       title: "Actions",
```

Comment:
Would highly recommend replacing with renderActionsColumn.

Reply:
This has been changed to use renderActionsColumn, thanks for the suggestion its a lot better with this.